content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -20,72 +20,61 @@ This page describes the RuntimeClass resource and runtime selection mechanism.
 
 ## Runtime Class
 
-RuntimeClass is an alpha feature for selecting the container runtime configuration to use to run a
-pod's containers.
+RuntimeClass is a beta feature (as of v1.14) for selecting the container runtime configuration to
+use to run a pod's containers.
 
-### Set Up
-
-As an early alpha feature, there are some additional setup steps that must be taken in order to use
-the RuntimeClass feature:
-
-1. Enable the RuntimeClass feature gate (on apiservers & kubelets, requires version 1.12+)
-2. Install the RuntimeClass CRD
-3. Configure the CRI implementation on nodes (runtime dependent)
-4. Create the corresponding RuntimeClass resources
+{{< warning >}} RuntimeClass includes *breaking* changes in the beta upgrade in v1.14. If you were
+using RuntimeClass prior to v1.14, the following actions are required:
 
-#### 1. Enable the RuntimeClass feature gate
+- The `node.k8s.io` API group and `runtimeclasses.node.k8s.io` resource have been migrated to a
+  built-in API. RuntimeClass resources must be recreated *after* upgrading to v1.14, and the
+  runtimeclasses.node.k8s.io CRD should be manually deleted.
+- The `spec.runtimeHandler` field in v1aplha1 has been renamed to `handler` (no more `spec`). The
+  handler is now a required field. Handlers can no longer include `.` characters.
 
-See [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/) for an explanation
-of enabling feature gates. The `RuntimeClass` feature gate must be enabled on apiservers _and_
-kubelets.
+{{< /warning >}}
 
-#### 2. Install the RuntimeClass CRD
-
-The RuntimeClass [CustomResourceDefinition][] (CRD) can be found in the addons directory of the
-Kubernetes git repo: [kubernetes/cluster/addons/runtimeclass/runtimeclass_crd.yaml][runtimeclass_crd]
-
-Install the CRD with `kubectl apply -f runtimeclass_crd.yaml`.
+### Set Up
 
-[CustomResourceDefinition]: /docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/
-[runtimeclass_crd]: https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/runtimeclass/runtimeclass_crd.yaml
+Prerequisite: Ensure the RuntimeClass feature gate is enabled (it is by default). See [Feature
+Gates](/docs/reference/command-line-tools-reference/feature-gates/) for an explanation of enabling
+feature gates. The `RuntimeClass` feature gate must be enabled on apiservers _and_ kubelets.
```

Comment:
nit: update the RuntimeClass feature gate to `beta`:
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

Reply:
Done